mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/domain/JobDefinition.java

```diff
@@ -246,6 +251,21 @@ public Optional<ClusterID> getResourceCluster() {
             .map(l -> ClusterID.of(l.getValue()));
     }
 
+    public Map<String, String> getAssignmentAttributes() {
+        final Pattern pattern = Pattern.compile(ALLOCATION_CONSTRAINT_LABEL_REGEX);
```

Comment:
nit: make this static.

Reply:
why?

Reply:
Why is the pattern compiled every time the assignment attributes are requested when the regex itself is static?

Reply:
Sorry I thought you were referring to the entire method. +1